py/selenium/webdriver/common/actions/wheel_input.py

```diff
@@ -21,6 +21,34 @@
 from selenium.webdriver.remote.webelement import WebElement
 
 
+class ScrollOrigin:
+
+    def __init__(self, origin: Union[str, WebElement], x_offset: int, y_offset: int) -> None:
+        self._origin = origin
+        self._x_offset = x_offset
+        self._y_offset = y_offset
+
+    @classmethod
+    def from_element(cls, element: WebElement, x_offset: int = 0, y_offset: int = 0):
```

Comment:
does this offer any value over the normal init? (can the dunder `__init__` have defaults for the offsets as `=0` itself?

Reply:
The idea is to make it explicit the object you get instead of just constructing by default. This is a common Java pattern that I also use in Ruby. If it isn't idiomatic Python we can adjust. Java can make the constructor private. Ruby & Python people can avoid the class methods, but we can at least have them for our documentation.